README.md

```diff
@@ -89,7 +89,6 @@ React group project
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 - [Live Demo Link](https://airpollutionapp.onrender.com/)
-Coming soon!
 
 ## Demonstration Video
 - [Demo-Video](https://www.loom.com/share/b53c268cca8b41adb0c7a3f4650cf8eb)
```

Comment:
You did a good job working on the project requirements. It would be nice if you recorded another video carefull showcasing your work and also show us a piece of code and a feature you are most proud of. Kindly base your video recording based on the following requirements.

Video presentation

Provide feedback about the video presentation in the code review summary.

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
@KayLemba  Done👌